apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/shard/helper/ValueParserImpl.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (c)  2018-2019. Apollo Foundation.
+ */
+
+package com.apollocurrency.aplwallet.apl.core.shard.helper;
+
+import com.apollocurrency.aplwallet.apl.core.shard.helper.csv.ValueParser;
+import com.apollocurrency.aplwallet.apl.util.StringUtils;
+
+import javax.inject.Singleton;
+import java.util.Base64;
+
+@Singleton
+public class ValueParserImpl implements ValueParser {
+    @Override
+    public String parseStringObject(Object data) {
+        String stringObject = (String)data;
+        String stringValue;
+        if (stringObject.charAt(0)=='\''){
+            if(stringObject.charAt(stringObject.length()-1)=='\'') {
+                stringValue = stringObject.substring(1, stringObject.length() - 1);
+            }else{
+                throw new RuntimeException("Wrong quotes balance: ["+stringObject+"]");
+            }
+        }else{//string without quotes
+            stringValue = stringObject;
+        }
+        String value = stringValue.replaceAll("''", "'");
+        return value;
+    }
+
+    @Override
+    public Object[] parseArrayObject(Object data) {
+        String objectArray = (String)data;
```

Comment:
NULL check would be safer for incoming param.

Reply:
done